Riot/ViewController/DeactivateAccount/DeactivateAccountViewController.m

```diff
@@ -0,0 +1,287 @@
+/*
+ Copyright 2018 New Vector Ltd
+ 
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+ 
+ http://www.apache.org/licenses/LICENSE-2.0
+ 
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+#import "DeactivateAccountViewController.h"
+
+#import "RiotDesignValues.h"
+
+#pragma mark - Defines & Constants
+
+static CGFloat const kButtonCornerRadius = 5.0;
+static CGFloat const kTextFontSize = 15.0;
+static NSTimeInterval const kActivityIndicatorAnimationDuration = 0.3;
+static CGFloat const kActivityContainerViewCornerRadius = 5.0;
+
+#pragma mark - Private Interface
+
+@interface DeactivateAccountViewController ()
+
+#pragma mark - Outlets
+
+@property (weak, nonatomic) IBOutlet UILabel *deactivateAccountInfosLabel;
+
+@property (weak, nonatomic) IBOutlet UILabel *forgetMessagesInfoLabel;
+@property (weak, nonatomic) IBOutlet UIButton *forgetMessageButton;
+
+@property (weak, nonatomic) IBOutlet UIButton *deactivateAcccountButton;
+
+
+#pragma mark - Private Properties
+
+@property (strong, nonatomic) NSDictionary *normalStringAttributes;
+@property (strong, nonatomic) NSDictionary *emphasizeStringAttributes;
+
+@property (strong, nonatomic) MXKErrorAlertPresentation *errorPresentation;
+
+@end
+
+#pragma mark - Implementation
+
+@implementation DeactivateAccountViewController
+
+#pragma mark - Setup & Teardown
+
++ (DeactivateAccountViewController*)instantiateWithMatrixSession:(MXSession*)matrixSession
+{
+   DeactivateAccountViewController* viewController = [[UIStoryboard storyboardWithName:NSStringFromClass([DeactivateAccountViewController class]) bundle:[NSBundle mainBundle]] instantiateInitialViewController];
+    [viewController addMatrixSession:matrixSession];
+    return viewController;
+}
+
+#pragma mark - View life cycle
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+    
+    self.title = NSLocalizedStringFromTable(@"deactivate_account_title", @"Vector", nil);
+    
+    self.errorPresentation = [[MXKErrorAlertPresentation alloc] init];
+    [self setupStringAttributes];
+    [self setupViews];
```

Comment:
You should observe the user interface theme change for this viewcontroller too.
(see for example: https://github.com/vector-im/riot-ios/blob/master/Riot/ViewController/AttachmentsViewController.m#L51)